midealocal/devices/__init__.py

```diff
@@ -16,10 +57,10 @@ def device_selector(
     customize: str,
 ) -> Any:
     try:
-        if device_type < 0xA0:
-            device_path = f".{'x%02x' % device_type}"
+        if device_type < DeviceType.A0:
+            device_path = f".{'x{:02x}'.format(device_type)}"

```

Comment:
The `format` seems related to UP031, what about fixing in a different PR ?

Reply:
sure